src/main/java/com/google/sps/servlets/DeleteOrganizationServlet.java

```diff
@@ -14,14 +14,29 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import com.google.sps.data.GivrUser;
-import java.io.IOException;
 
+/** Servlet responsible for deleting tasks. */
 @WebServlet("/delete-organization")
 public class DeleteOrganizationServlet extends HttpServlet {
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long id = Long.parseLong(request.getParameter("id"));
+
+    Key DistributorEntityKey = KeyFactory.createKey("Distributor", id);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.delete(DistributorEntityKey);
+
```

Comment:
When an organization gets deleted from the datastore, should that organization's moderators be deleted from the user table?

Reply:
Existence in the user table is unrelated to whether they're a moderator right? (So we should avoid modifying the user table based on changes to moderation status)

Reply:
Yes we decided to just call the update moderating org function for each user in the moderator list (latest commit)